AI-and-Analytics/End-to-end-Workloads/Census/README.md

```diff
@@ -74,23 +74,28 @@ To learn more about the extensions and how to configure the oneAPI environment,
 
 ### On Linux*
 
-1. Install the Intel® Distribution of Modin* python environment.
+1. Install the Intel® Distribution of Modin* python environment (Only python 3.8 - 3.10 are supported).
    ```
-   conda create -y -n intel-aikit-modin intel-aikit-modin -c intel
+   conda create -n modin-hdk python=3.x -y
    ```
 2. Activate the Conda environment.
    ```
-   conda activate intel-aikit-modin
+   conda activate modin-hdk
    ```
-3. Install Jupyter Notebook.
+3. Install Intel® Extension for Scikit-learn* and related libraries.
    ```
-   conda install jupyter nb_conda_kernels
+   pip install scikit-learn-intelex
```

Comment:
why couldn't we use scikit-learn from AI kit base env?

Reply:
Do you mean clone an environment from AI kit base env?